src/software/thunderscope/replay/proto_player.py

```diff
@@ -85,22 +109,138 @@ def __sort_replay_chunks(file_path: str):
             replay_index, _ = tail.split(".")
             return int(replay_index)
 
-        self.sorted_chunks = sorted(replay_files, key=__sort_replay_chunks)
+        return sorted(replay_files, key=__sort_replay_chunks)
+
+    @staticmethod
+    def is_valid_protobuf(message):
```

Comment:
can you describe why this is necessary in the code itself?

it's good to describe reasoning in the code in case someone needs to go and look at the code

Reply:
deleted code.

Reply:
looks like it isn't deleted -- maybe you forgot to commit?